classes/MyTableLister.php

```diff
@@ -535,22 +559,102 @@ public function resolveSQL($sql, $success, $error)
     /**
      * Attempt to a user-defined translation
      * @param string $text
-     * @param bool $escape escape for HTML (ie. tranfer ' " <… to &#39; &quot; &lt;…) ?
+     * @param bool $escape escape for HTML?
      * 
      * @return string
      */
     public function translate($text, $escape = true)
     {
-        //@todo kdy může nastat situace, že TableListerTranslate neexistuje? Proč to nenadefinovat jako GodsDev\mycms\Utils\TableListerTranslate ?
-        if (function_exists('TableListerTranslate')) {
-            $result = TableListerTranslate($text);
-        } else {
-            $result = $text;
+        if (isset($this->TRANSLATION[$text])) {
+            $text = $this->TRANSLATION[$text];
         }
         if ($escape) {
-            $result = Tools::h($result);
+            $text = Tools::h($text);
         }
-        return $result;
+        return $text;
```

Comment:
co to zkrátit na
return $escape?Tools::h($text):$text;

Reply:
Jo, to by šlo.